ford_fusion_sehybrid_2019/docker-compose.yml

```diff
@@ -44,6 +44,19 @@ services:
       - /opt/carma/maps:/opt/carma/maps
       - /opt/carma/routes:/opt/carma/routes
     command: wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/carma_docker.launch
+    
+  lightbar-driver:
+    image: usdotfhwastol/carma-lightbar-driver:latest
+    network_mode: host
+    container_name: lightbar-driver
+    volumes_from: 
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /home/autoware/.ros:/home/carma/.ros
+    environment:
+      - ROS_IP=127.0.0.1
```

Comment:
This is technically correct, but set it to - ROS_IP=192.168.88.10
To be consistent with the rest of the file.